helm-charts/iqserver/templates/deployment.yaml

```diff
@@ -52,13 +52,36 @@ spec:
             name: nxiq-pv-log
           - mountPath: /etc/nexus-iq-server
             name: config-volume
+          {{- if  .Values.iq.certificates }}
+          - mountPath: /etc/nexus-iq-server/jks
+            name: nxiq-pv-jks
+          {{- end }}
           {{- if .Values.iq.licenseSecret }}
           - mountPath: /etc/nexus-iq-license
             name: license-volume
           {{- end }}
-      {{- if .Values.initContainers }}
+      {{- if (or .Values.initContainers .Values.iq.certificates) }}
       initContainers:
-        {{- toYaml .Values.initContainers | nindent 8 }}
+        {{- if .Values.iq.certificates }}
+        - name: build-jks
+          image: {{ .Values.iq.imageName }}
```

Comment:
use the same IQ image, since it has java already, and i don't need to think about introducing new images and what that does to certification.